### PR TITLE
feat: set up loader and connection to query partner artists wit…

### PR DESCRIPTION
**Pull request by jpotts244**
…h alert counts from gravity

Part 2 of 3 to complete this feature: https://artsyproduct.atlassian.net/browse/AMBER-768
And exposed new gravity endpoint defined [here](https://github.com/artsy/gravity/pull/18000)

Example query structure:
```graphql
{
  partner(id: "commerce-test-partner") {
    slug
    name
    artistsWithAlertCountsConnection(first:8, sort: SORTABLE_ID_DESC) {
      totalCount
      edges {
        totalAlertCount
        node {
          name
          imageUrl
          
        }
      }
    }
  }
}

```

<summary> Example response shape
<details> 

```
{
  "data": {
    "partner": {
      "slug": "commerce-test-partner",
      "name": "Commerce Test Partner",
      "artistsWithAlertCountsConnection": {
        "totalCount": 20,
        "edges": [
          {
            "totalAlertCount": 0,
            "node": {
              "name": "Jon Allured",
              "imageUrl": null
            }
          },
          {
            "totalAlertCount": 0,
            "node": {
              "name": "Gabriel de la Mora",
              "imageUrl": "https://d32dm0rphc51dk.cloudfront.net/4TR6sjESMxV_8MSXoxjzEw/square.jpg"
            }
          },
          {
            "totalAlertCount": 20,
            "node": {
              "name": "Damien Hirst",
              "imageUrl": "https://d32dm0rphc51dk.cloudfront.net/TLj5ypujA8_sBmcjWw6cRw/square.jpg"
            }
          },
          {
            "totalAlertCount": 0,
            "node": {
              "name": "John John",
              "imageUrl": null
            }
          },
          {
            "totalAlertCount": 0,
            "node": {
              "name": "Karen Knorr",
              "imageUrl": "https://d32dm0rphc51dk.cloudfront.net/ia5hSauv4IESKA0uyh1F_Q/square.jpg"
            }
          },
          {
            "totalAlertCount": 0,
            "node": {
              "name": "Molly Magai",
              "imageUrl": "https://d32dm0rphc51dk.cloudfront.net/04x1b4qE3LpJeFL5d7eB6Q/square.jpg"
            }
          },
          {
            "totalAlertCount": 0,
            "node": {
              "name": "Mike Mandel",
              "imageUrl": "https://d32dm0rphc51dk.cloudfront.net/vlGHR3u-JYpd3fzoeUpBzg/square.jpg"
            }
          },
          {
            "totalAlertCount": 0,
            "node": {
              "name": "Testy McTest",
              "imageUrl": "https://d32dm0rphc51dk.cloudfront.net/ocLcYX0Yy7RKSbqi1zIVMg/square.jpg"
            }
          }
        ]
      }
    }
  },
  "extensions": {
    "requests": {
      "gravity": {
        "requests": {
          "partner/commerce-test-partner": {
            "time": "0s 2.806ms",
            "cache": true,
            "length": "N/A"
          },
          "partner/5f80bfefe8d808000ea212c1/artists_with_alert_counts?page=1&size=8&total_count=true": {
            "time": "0s 187.876ms",
            "cache": false,
            "length": "0 B"
          }
        }
      }
    },
    "requestID": "7ef071e0-5354-11ef-b18f-6f6cbea138c5",
    "userAgent": "graphiql-app"
  }
}
```

</details>
</summary>

---

Allows for sorting by A-Z/Z-A of artist names:

Screenshot 2024-08-05 at 2 47 50 PM
Screenshot 2024-08-05 at 2 47 32 PM

### CODE REVIEW

File: src/schema/v2/partner/partner.ts [hunk 1]
```diff
@@ -233,6 +246,54 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           })
         },
       },
+      artistsWithAlertCounts: {
```
Comment (damassi) [inline src/schema/v2/partner/partner.ts:249]: If this is a connection we should make it known in the name! So, `artistsWithAlertCountsConnection`

Comment (jpotts244) [reply src/schema/v2/partner/partner.ts:263]: Excellent eyes 🦅 !

**Review by damassi — APPROVED** [review]
Looks great 👍 (tho it looks like you maybe skipped the git precommit hooks?)

**Comment by jpotts244** [comment]
Re-pushed a few times!!
Was having wifi issues, I think the pre-commit / pre-push hooks should be up to date now

**Pull request merged**